main.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"github.com/jfrog/build-info-go/commands"
+	"github.com/jfrog/build-info-go/utils"
+	"github.com/urfave/cli/v2"
+	"os"
+)
+
+var log utils.Log
+
+func main() {
+	log = utils.NewDefaultLogger(getCliLogLevel())
+	app := &cli.App{
+		Name:     "JFrog Build-Info CLI",
+		Usage:    "create and collect build-info for your projects",
```

Comment:
create and collect build-info for your projects
-->
Generate build-info for your source code